HOW_TO_BUILD.md

```diff
@@ -23,12 +23,12 @@ To build the Torpedo container without deploying: `make container`
 #### To build a specific ginkgo test
 
 ```
-ginkgo build  <path_to_test_pkg>
+ginkgo build <path_to_test_pkg>
```

Comment:
Hey Paul, thanks for updating the doc!
I found that, after running ginkgo build <path_to_test_pkg>, the test is only updated under the <path_to_test_pkg>, not under the bin/ dir. That confused me before, because HOW_TO_RUN.md tells me to run the test under bin/, which might be updated only when we build the project with the make commands. 
So I am wondering if we should indicate that here.

Reply:
make clean && make build  should put it in the bin folder